packages/react/src/components/Table/TableBody/TableBodyLoadMoreRow/TableBodyLoadMoreRow.jsx

```diff
@@ -35,21 +35,29 @@ const TableBodyLoadMoreRow = ({
 }) => {
   return (
     <TableRow isSelected={false}>
-      <TableCell
-        key={`${tableId}-${id}-row-load-more-cell`}
-        colSpan={totalColumns}
-        className={`${iotPrefix}--load-more-cell`}
-        data-testid={`${testId}-${id}-load-more`}
-      >
-        <Button
-          className={`${iotPrefix}--load-more-cell--content`}
-          onClick={() => onRowLoadMore(id)}
-          kind="ghost"
-          loading={isLoadingMore}
+      {isLoadingMore ? (
+        Array.from(Array(totalColumns)).map((_, index) => (
```

Comment:
I don't this is gonna work as expected. TotalColumns was great for creating one big column using colspan but now we are going to have "skeletons" in all the cells, not only the actual data cells. 

You need to know how many "extra" columns the row has before and after the actual data columns. You can have a look to see how it is handled in the `TableBodyRowRenderer.jsx`